chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -398,6 +398,37 @@
   end
 end
 
+include_recipe "apache2"
+
+apache_module "proxy" do
+  conf true
+end
+
+apache_module "proxy_balancer" do
+  conf false
+end
+
+apache_module "proxy_http" do
+  conf false
+end
+
+apache_module "rewrite" do
+  conf false
+end
+
+template "#{node[:apache][:dir]}/vhosts.d/crowbar.conf" do
+  source "apache.conf.erb"
+  mode 0644
+
+  variables(
+    port: node["crowbar"]["web_port"] || 3000,
+    logfile: "/var/log/apache2/crowbar-access_log",
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.